code/src/components/ThoughtInput.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+
+const ThoughtInput = ({ onThoughtChange }) => {
+    const [newThought, setNewThought] = useState('');
+    
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        onThoughtChange(newThought);
+    }
+
+    return (
+        <form className="post" onSubmit={handleSubmit}>
+            <label 
+                className="post-header" 
+                htmlFor="post"
+            >
+                What's making you happy right now?
+            </label>
+            <input
+                type="text"
+                id="post"
+                onChange={event => setNewThought(event.target.value)}
```

Comment:
You can do some improvements like to add value={newThought} and then on line 9 you can clear the newThought setNewThought(""). When you submit the thought it will be reset to empty